css-animations-1/animation-duration-005.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Animations Test: animation-duration - 0s</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<link rel="help" href="http://dev.w3.org/csswg/css-animations-1/#animation-name">
+<link rel="help" href="http://dev.w3.org/csswg/css-animations-1/#animation-duration">
+<link rel="help" href="http://dev.w3.org/csswg/css-animations-1/#animation-fill-mode">
+<meta name="flags" content="animated">
+<meta name="assert" content="When animation-duration is set to 0s (zero seconds),
+                             and 'animation-fill-mode' is set to 'forwards' or 'both',
+                             the last frame of the animation will be displayed.">
+<style>
+  div {
+    height: 100px;
+    width: 100px;
+    position: relative;
+  }
+
+  #test {
+    animation-name: sample;
+    animation-duration: 0s;
+    animation-fill-mode: forwards;
+
+    background-color: blue;
+  }
+
+  #ref {
+    background-color: yellow;
+    left: 150px;
+  }
+
+  @keyframes sample {
+    from {
+      left: 0px;
+    }
+    to {
+      left: 150px;
+    }
+  }
+</style>
+<body>
+  <p>
+    Test passes if there is a filled blue with 'Filler Text'
```

Comment:
To make the pass condition easier, we can have `div#height` 50px. Then check that there is a filled blue square with 'Filler Text'.